app/scripts/store/dimFarming.directive.js

```diff
@@ -16,9 +16,9 @@
             <img class="engram" ng-class="{ active: (vm.service.movingItems || vm.service.makingRoom) }" src="/images/engram.svg" height="60" width="60"/>
           </span>
           <span>
-            <p>DIM is moving Engram and Glimmer items from {{vm.service.store.name}} to the vault and leaving one space open per item type to prevent anything from going to the Postmaster.</p> 
+            <p>DIM is moving Engram and Glimmer items from {{vm.service.store.name}} to the vault and leaving one space open per item type to prevent anything from going to the Postmaster.</p>
             <div class="item-details"><span>
-              <p><span translate="configuration"></span></p>
```

Comment:
I mean, maybe even `<p translate="configuration"></p>`... but again, don't worry about changing it. Was more of an open-air question/statement since we're doing this app wide.